apps/server/src/infra/authorization-client/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './authorization-client.adapter';
```

Comment:
In module barrel files bitte immer explizite exports anstat wildcards verwenden.

export { AuthorizationClientAdapter } from './authorization-client.adapter';
export { AuthorizationClientModule } from './authorization-client.module';